blackduck-security-task/test/unit/blackduck-security-task/bridge.spec.ts

```diff
@@ -261,20 +261,30 @@ describe("Download Bridge", () => {
     const cpuInfo = os.cpus()
     const isIntel = cpuInfo[0].model.includes("Intel")
     let bridgeDefaultPath = "";
+    let bridgeExecutablePath = "";
+    let bridgeSubDir = "";
     if (osName === "linux") {
-        bridgeDefaultPath = path.join(process.env["HOME"] as string, constants.BRIDGE_CLI_DEFAULT_PATH_LINUX);
-        bridgeUrl = "https://repo.blackduck.com/bds-integrations-release/com/blackduck/integration/bridge-cli/0.1.244/bridge-cli-0.1.244-linux64.zip"
+        bridgeDefaultPath = path.join(process.env["HOME"] as string, constants.BRIDGE_CLI_DEFAULT_PATH_UNIX);
+        bridgeExecutablePath = bridgeDefaultPath.concat("/bridge-cli-bundle-linux64")
+        bridgeSubDir = "/bridge-cli-bundle-linux64";
+        bridgeUrl = "https://artifactory.internal.synopsys.com/artifactory/clops-local/clops.sig.synopsys.com/bridge/binaries/bridge-cli-bundle/2.9.2/bridge-cli-bundle-2.9.2-linux64.zip"
     } else if (osName === "win32") {
```

Comment:
use constants for 'darwin', 'linux' and 'win32' as it is used in multiple places

Reply:
updated with constants